src/node_os.cc

```diff
@@ -360,6 +360,7 @@ static void GetUserInfo(const FunctionCallbackInfo<Value>& args) {
   }
 
   const int err = uv_os_get_passwd(&pwd);
+  OnScopeLeave free_passwd([&]() { uv_os_free_passwd(&pwd); });
```

Comment:
is it safe to call it if `uv_os_get_passwd` returned an error?

Reply:
Yeah, I think you’re right, we shouldn’t do that. I’ve moved it below the conditional :+1: